sites/mosul/configs/openmrs/initializer_config/ampathforms/F14-Basic_obstetric_ultrasound.json

```diff
@@ -648,7 +642,7 @@
                 ]
               },
               "hide": {
-                "hideWhenExpression": "isEmpty(fetus_numberOfFetuses) || fetus_numberOfFetuses !== '9c407911-f329-4761-b27a-bbbe24cc0332' && fetus_numberOfFetuses !== '42965304-8abb-4a57-b7c2-b3eff5cbe9c7' && fetus_numberOfFetuses !== '168e35f0-9866-4b13-98f2-4ecd2f538117' && fetus_numberOfFetuses !== '6bb1c6b4-3fa6-40ac-a7c6-7018bb07fd91'"
```

Comment:
Removing `isEmpty` conditions, since they don't add any advantage to the hide logics. The hide logics consists of the `!==` with concept uuids and `undefined`/`null` isn't equal to the concept uuids in any check.